### PR TITLE
ControlHub+ Absolute Positioning

### PR DESCRIPTION
**Pull request by carlalldis**
Added absolute positioning functions for Control Hub Plus:

setMotorAbsolutePosition
setMotorEncoderPosition

It would be good to have a getMotorAbsolutePosition function, but I do not understand how to query this via the base library; I believe this would be a [Port Mode Information Request](https://lego.github.io/lego-ble-wireless-protocol-docs/index.html#port-mode-information-request) for the device's SI value (0x03).

This was tested with a 88013 Technic Large Motor.

### CODE REVIEW

**Comment by corneliusmunz** [comment]
Hi @carlalldis! Many thanks for your PR. I will have a look on it and i think it makes definitely sense. Maybe i will suggest to add the two additional commands provided by your PR to the lpf2hub base class. Becaus the commands could also be used with the Boost or PoweredUp Hubs with an Tacho motor plugged in. About the getMotorAbsolutePostion: This should already be working. Maybe you can have a look in the following example: https://github.com/corneliusmunz/legoino/blob/master/examples/BoostHubRotationSensor/BoostHubRotationSensor.ino I am currently working on generic callback functions. This is much better than in the example above but this will need some time till i have a tested version.

So again: Many thanks for that contribution

**Review by corneliusmunz — CHANGES_REQUESTED** [review]
I have checked the requested changes and it works fine. I like the example sketch with calibrating the steering. Please change the filename of the example to the same name as the folder. Then i will merge your PR directly in the develop branch. I will later on refactor the motor functions to the main class because the functions are now duplicated for the Boost and ControlPlus Hub. The functions are not really dependent on the hub, they are dependent on the type of motor which is connected to a port.

File: examples/ControlPlusHubAbsolute/ControlHubPlusAbsolute.ino [hunk 1]
```diff
@@ -0,0 +1,80 @@
+/**
```
[the remark this answers is not in the document]
Comment (carlalldis) [reply examples/ControlPlusHubAbsolute/ControlHubPlusAbsolute.ino]: Done, nicely spotted - took me a few moments to see what was wrong.

**Review by corneliusmunz — APPROVED** [review]
Super! Thanks for that really fast change :-) I will merge it and it will be part of the next release

**Comment by carlalldis** [comment]
No problem; thanks for all the comments. I had a look at getTachoMotorRotation() before but thought it might not work with Control+ as it didn't appear to have a port parameter to specify which Motor to read from. I've looked at the example again and think I understand now about activating the port device first - I will see if this works. Thanks!

**Pull request merged**